### PR TITLE
make source more pretty

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
btw, I'ld like to finish and merge this, but it should be done at a time with not many parallel open PRs to avoid conflicts.

File: src/borg/archive.py [hunk 1]
```diff
@@ -1215,52 +1216,53 @@ def valid_item(obj):
...
-            archive_items = [item for item in self.manifest.archives.items() if item[0] == archive]
-            if not archive_items:
+            info = self.manifest.archives.get(archive)
+            if info is None:
                 logger.error("Archive '%s' not found.", archive)
+                archive_infos = []
+            else:
+                archive_infos = [info]
             num_archives = 1
             end = 1
```
Comment (ThomasWaldmann) [inline src/borg/archive.py]: these "= 1" might be wrong if the archive is not found, need to check this.

**Comment by ThomasWaldmann** [comment]
merge? anything else to clean up that way?

File: src/borg/archive.py (hunk 1 above)
Comment (enkore) [inline src/borg/archive.py]: the whole section looks like it could use a minor refactoring.

**Comment by enkore** [comment]
should be it

**Pull request merged**